doc/classes/Animation.xml

```diff
@@ -319,6 +321,7 @@
 			<argument index="1" name="enabled" type="bool">
 			</argument>
 			<description>
+				If [code]enabled[/code] is [code]true[/code], the track at index [code]idx[/code] is enabled. Otherwise it will be disabled. Tracks are enabled by default.
```

Comment:
"Otherwise it will be disabled." - Avoid future tense, stick to present.

This seems overly wordy to me. Would it be too terse to just say:

    Enables/disables the given track. Tracks are enabled by default.

Reply:
I was under the impression we were supposed to be including the parameter's name in the description to show directly how it is involved. Is stuff like that ^ acceptable (cause I'd prefer it that way)?

Reply:
It's not unacceptable, and I do tend to err on the side of terseness. The right answer is probably somewhere in the middle. :)